src/services/CompaniesService.js

```diff
@@ -0,0 +1,31 @@
+import http from './HttpService';
+
+const company = {
+    createCompany: async function () {
+        let token = localStorage.getItem('token');
+        const company = await http.post(
+            '/companies',
+            {
+                data: {
+                    name: 'quantox',
+                },
+            },
+            {
+                headers: {
+                    Authorization: `Bearer ${token}`,
+                },
```

Comment:
This should be done in Http service once, when we're setting up axios , so we don't have to write it for every ajax call in the application :)